test-poc/base/src/test/java/org/keycloak/test/base/ManagedResources2Test.java

```diff
@@ -0,0 +1,37 @@
+package org.keycloak.test.base;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.keycloak.admin.client.resource.ClientResource;
+import org.keycloak.admin.client.resource.RealmResource;
+import org.keycloak.representations.idm.ClientRepresentation;
+import org.keycloak.test.framework.KeycloakIntegrationTest;
+import org.keycloak.test.framework.TestClient;
+import org.keycloak.test.framework.TestRealm;
+import org.keycloak.test.framework.injection.LifeCycle;
+
+import java.util.List;
+
+@KeycloakIntegrationTest
+public class ManagedResources2Test {
+
+    @TestRealm(lifecycle = LifeCycle.CLASS)
```

Comment:
Looks like the default lifecycle is already `CLASS` so this can probably be omitted.
```suggestion
    @TestRealm
```